pkg/config/log.go

```diff
@@ -20,8 +20,19 @@ import (
 	"github.com/cihub/seelog"
 )
 
+// LoggerName is a string prepended to all logs to indicate which component is logging
+type LoggerName string
+
 const logFileMaxSize = 10 * 1024 * 1024         // 10MB
 const logDateFormat = "2006-01-02 15:04:05 MST" // see time.Format for format syntax
+// CoreLogger is the name of the core agent logger
+const CoreLogger LoggerName = "CORE"
+
+// DCALogger is the name of the cluster agent logger
+const DCALogger LoggerName = "DCA"
```

Comment:
Lastly, these constants seem like they belong in their own section of the code (where `SetupLogger` is called). What I'm saying is:
* `DCALogger` should simply be called `const loggerName = "DCA"` inside the cluster-agent package
* `TRACE` should simply be called `const loggerName = "DCA"` inside the trace-agent package
* etc...

Reply:
I moved the const in the cmd of each agent, WDYT?

Reply:
Thanks! I think this is possibly better, since each one of us has control over their own names. I'll resolve this comment now.